rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book, :date
```

Comment:
Great job @Roland-Ntwali 🤝

- [ ] As the previous reviewer requested, kindly use `attr_accessor` to set both the getter and setter.